exp-player/addon/components/exp-exit-survey.js

```diff
@@ -148,7 +147,7 @@ export default ExpFrameBaseComponent.extend({
                 Ember.merge(formData, this.getValue());
                 root.set('formData', formData);
                 console.log('Post-study survey complete.');
-                root.actions.next();
+                root.actions.next.apply(root);
```

Comment:
Interesting- it's because `formActions` aren't actions at all. It's a lightly documented hash of functions tied to buttons. (the guy behind dynamic forms gave a talk recently and acknowledged that he hadn't put full attention into documentation just yet)

That could be gnarly indeed; we can revisit this later when we have more time.
https://toddjordan.github.io/ember-cli-dynamic-forms/#/demos/actions